shared/js/content-scope/tracking-cookies-3p-protection.js

```diff
@@ -1,18 +1,39 @@
-import { defineProperty } from './utils'
+import { defineProperty, postDebugMessage } from './utils'
 
-function blockCookies () {
+function blockCookies (debug) {
     // disable setting cookies
     defineProperty(document, 'cookie', {
         configurable: false,
-        set: function (value) { },
-        get: () => ''
+        set: function (value) {
+            if (debug) {
+                postDebugMessage('jscookie', {
+                    action: 'block',
+                    reason: 'tracker frame',
+                    documentUrl: document.location.href,
+                    scriptOrigins: [],
+                    value: value
+                })
```

Comment:
this is just debug for manual debugging? Or are we depending on those messages in tests somehow?

Reply:
These are for the debugger panel, yes.